.changeset/chilly-ties-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: false positives for "never" in `selector-attribute-quotes`
```

Comment:
[suggest] Can you please make the format consistent by following the current guideline? (I've changed the PR title also)

https://github.com/stylelint/stylelint/blob/005ed45ff170f3e3351a610c6cdd8931f6731f49/docs/maintainer-guide/issues.md?plain=1#L41

```suggestion
Fixed: `selector-attribute-quotes` false positives for "never"
```